Update interfaces/SVG.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/0b25b9f/whatwg/idl/SVG.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/477693484

foolip commented:
https://github.com/tidoust/reffy/issues/133 is at play again, but now there are some other change too. @tidoust, have you had an opportunity to investigate why this bit of IDL is flipping back and forth?

autofoolip committed:
Update interfaces/SVG.idl

Source: https://github.com/tidoust/reffy-reports/blob/0b25b9f/whatwg/idl/SVG.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/477693484

tidoust commented:
@foolip, I still don't have any rational explanation, but why is this one invalid? The IDL changes here match changes in the SVG spec 10 days ago in particular: https://github.com/w3c/svgwg/commit/d534877e5a0a83000c0dea391914248140a34be8

foolip commented:
@tidoust between your comment and mine, the bot updated the PR to not include the bad bit. So it's still flipping back and forth. I'll merge this while it's still in the good state :)
